src/main/java/io/vertx/cassandra/impl/CassandraClientImpl.java

```diff
@@ -98,6 +100,25 @@ public CassandraClient connect(String keyspace, Handler<AsyncResult<Void>> conne
     return this;
   }
 
+  @Override
+  public CassandraClient executeWithFullFetch(String query, Handler<AsyncResult<List<Row>>> resultHandler) {
+    return executeWithFullFetch(new SimpleStatement(query), resultHandler);
+  }
+
+  @Override
+  public CassandraClient executeWithFullFetch(Statement statement, Handler<AsyncResult<List<Row>>> resultHandler) {
+    execute(statement, done -> {
```

Comment:
Minor: you could simplify with Future composition. But it's fine to keep it as-is.

Reply:
I would prefer to keep it as-is

Reply:
I mean:

```java
Future<ResultSet> resultSetFuture = Future.future();
execute(statement, resultSetFuture);
resultSetFuture.compose(resultSet-> {
  Future<List<Row>> rowsFuture = Future.future();
  resultSet.all(rowsFuture);
  return rowsFuture;
}).setHandler(resultHandler);
```

Reply:
Done